style.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+  text-decoration: none;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+@media only screen and (max-width: 375px),
+  (max-width: 345px),
+  (max-width: 390px),
+  (max-width: 414px) {
+  body {
+    background: #e5e5e5;
+    margin: 0 auto;
+    padding: 0 auto;
+    text-align: center;
+  }
```

Comment:
- [ ] As requested by the previous reviewer you should stick to the design as much as possible like color, background, font, etc. As I notice your design is broken so the mobile version should be responsive until the desktop screen size that 768px. To fix this issue please use `max-width: 768px`.
### Your design
![his](https://user-images.githubusercontent.com/81727988/234654994-63e1655b-e4c1-4e04-9226-85c117a60314.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/234655026-fd674742-91b4-4e5a-aa6f-19049b73f02d.png)